docs/job-scheduling.md

```diff
@@ -215,6 +215,9 @@ pool), but inside each pool, jobs run in FIFO order. For example, if you create
 means that each user will get an equal share of the cluster, and that each user's queries will run in
 order instead of later queries taking resources from that user's earlier ones.
 
+If jobs are not explicitely set to use a given pool, they end up in the default pool. This means that even if
```

Comment:
Hi @Alexis-D , there are a few minor typos here;
 'explicitely' -> 'explicitly'.
'ran' -> 'run'

Reply:
right my bad -- I updated the PR